pkg/restapi/rp/operation/operations.go

```diff
@@ -696,9 +697,24 @@ func (o *Operation) getPresentationsRequest(w http.ResponseWriter, r *http.Reque
 		return
 	}
 
+	var governanceVC []byte
+
+	if o.governanceProvider != nil {
+		var err error
+		governanceVC, err = o.governanceProvider.GetCredential(cr.CR.GetPayload().Client.ClientID)
```

Comment:
Should be the RP's publicDID

Reply:
we retrieve by profile ID check the code

Reply:
@llorllale https://github.com/trustbloc/edge-adapter/blob/master/pkg/governance/provider.go#L142

Reply:
This will need to be revised. There is a 1:1 relation between the RP's public DID and their clientID.